lib/Parser/Parse.cpp

```diff
@@ -2447,6 +2523,146 @@ IdentPtr Parser::ParseImportOrExportFromClause(bool throwIfNotFound)
     return moduleSpecifier;
 }
 
+template<bool buildAST> 
+ParseNodePtr Parser::ParseDefaultExportClause()
+{
+    Assert(m_token.tk == tkDEFAULT);
+
+    m_pscan->Scan();
+    ParseNodePtr pnode = nullptr;
+    ushort flags = fFncNoFlgs;
+    
+    switch (m_token.tk)
+    {
+    case tkCLASS:
+        {
+            if (!m_scriptContext->GetConfig()->IsES6ClassAndExtendsEnabled())
+            {
+                goto LDefault;
```

Comment:
@abchatra felt it was too early to remove support for disabling the class feature.  I disagree but cannot defend my opinion with strong reason to remove it.  It _might_ be beneficial in the future to help reduce website bugs because of eval of classes.  Seems weak, but removing it just for aesthetic and a little less distraction in the code doesn't persuade us enough to make the call on it.